### PR TITLE
Add time measurement flag to all elf binaries in Makefile

### PR DESCRIPTION
**Pull request by tk26eng**
## What this patch does to fix the issue.
This patch makes code simpler.

## Link to any relevant issues or pull requests.
#924

### CODE REVIEW

File: blueoil/converter/templates/Makefile [hunk 1]
```diff
@@ -116,19 +116,19 @@ clean:
 	-$(RM) $(OBJ)
 
 lm_x86:           CXX = g++
-lm_x86:           FLAGS += $(INCLUDES) -O3 -std=c++14 -DUSE_PNG -pthread -g
+lm_x86:           FLAGS += $(INCLUDES) -O3 -std=c++14 -DUSE_PNG -pthread -g -DFUNC_TIME_MEASUREMENT
```
Comment (tsawada) [inline blueoil/converter/templates/Makefile]: maybe we should place all `-D` flags next to each other.
`-DUSE_PNG -DFUNC_TIME_MEASUREMENT -pthread -g`

Comment (tk26eng) [reply blueoil/converter/templates/Makefile]: Maybe it's better to place `-D` like you pointed out.
That will be modified in another PR.

**Comment by tk26eng** [comment]
Fix #924

**Pull request merged**